### PR TITLE
[9.0][PORT] base_partner_merge

### PR DESCRIPTION
**Pull request by yvaucher**
Migration of module base_partner_merge.

It proposes a simple way allows to use the crm deduplicate wizard without having to install CRM module.

See the discution here: https://github.com/OCA/partner-contact/issues/196#issuecomment-169781002

Thanks @hbrunn for the idea.

I set `_module = 'base_partner_merge'` to ensure the model are loaded.

I will let you do other PRs to extend or modify the behaviour of the wizard if needed to alter the way the official wizard works.

### CODE REVIEW

File: base_partner_merge/__openerp__.py [hunk 1]
```diff
@@ -1,18 +1,16 @@
...
-    'author': "OpenERP S.A.,Odoo Community Association (OCA)",
+    'name': 'Deduplicate Contacts (No CRM)',
+    'summary': 'Partner merge wizard without dependency on CRM',
+    'author': "Camptocamp,Odoo Community Association (OCA)",
     'category': 'Generic Modules/Base',
-    'version': '8.0.0.1.0',
-    'description': """
-backport module, to be removed when we switch to saas2 on the private servers
-""",
+    'version': '9.0.0.1.0',
```
Comment (pedrobaeza) [inline base_partner_merge/__openerp__.py]: 9.0.1.0.0

File: base_partner_merge/base_partner_merge.py [hunk 2]
```diff
@@ -1,895 +1,11 @@
-# -*- coding: utf-8 -*-
+from openerp.addons.crm.base_partner_merge import *
```
Comment (pedrobaeza) [inline base_partner_merge/base_partner_merge.py]: Put a try here if there are custom deployments without the crm module.

Comment (pedrobaeza) [inline base_partner_merge/base_partner_merge.py]: Can you also be more specific about what to import instead of using `*`?

Comment (yvaucher) [inline base_partner_merge/base_partner_merge.py]: @pedrobaeza I want the whole file to simulate it is there.

Comment (pedrobaeza) [inline base_partner_merge/base_partner_merge.py]: OK then

**Comment by pedrobaeza** [comment]
Please bump the version and check Travis.

**Comment by yvaucher** [comment]
Fixed

**Comment by hbrunn** [comment]
cool that this works! I pondered if it's a good idea to load the view from the crm addon to in a function called from xml files, but that's a bit tricky and we wouldn't be able to add extra features, so I think this is fine the way it is :+1:

**Comment by lmignon** [comment]
:+1:  Works like a charm. Thanh you @pedrobaeza

**Comment by yvaucher** [comment]
@lmignon wrong PR? Or didn't got your tee/coffee yet? :smile:

**Comment by yvaucher** [comment]
Thanks for the merge @pedrobaeza

**Comment by lmignon** [comment]
oupssss not the wrong PR my thankfulness is for you @yvaucher

**Pull request merged**